### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by peelerzin**
Please review it im not sure about the routing siblings.

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -1,7 +1,79 @@
...
+  },
+
+  {
+    element: <Ticket />,
+    path: "ticket",
+  },
+
+  {
+    element: <RootLayout />,
+    path: "layout",
```
Comment (ReallyBoard123) [inline src/main.tsx:61]: where is /layout going to?

File: src/main.tsx [hunk 2]
```diff
@@ -1,7 +1,79 @@
...
+    element: <Login />,
+    index: true,
+  },
+
+  {
+    element: <Movies />,
+    path: "movies",
+  },
+
+  {
```
Comment (ReallyBoard123) [inline src/main.tsx:34]: from moviedetail to ticket, they could all be children of /movie, and you can use routes like:

```tsx
 {
        path: "blog",
        element: <BlogLayout />,
        children: [
          {
            path: ":postId",
            element: <Post />,
          },  {
            path: ":postId/edit",
            element: <EditPost />,
          },


```

File: src/layouts/MovieDetailLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { NavLink, Outlet } from "react-router-dom";
+
+const routesMovieDetail = [
```
Comment (ReallyBoard123) [inline src/layouts/MovieDetailLayout.tsx:3]: where are you using these bindings?

File: src/layouts/MovieDetailLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    path: "/ticket",
+  },
+];
+
+export function MovieDetailLayout() {
+  return (
+    <div>
+      <h1>Movies</h1>
+
+      {/* <nav>
```
Comment (ReallyBoard123) [inline src/layouts/MovieDetailLayout.tsx:31]: try to see if you can resolve all these uncommented codes

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
Posted comments in the given places. See that you use a layout-based navigation. Perhaps, reconsider your decision of having login as the route "/" . 
- Your login could live on /login as one top-level route.
- Then you have your navigation layout that appear on 2 screens? (Home, Movies, feauture1?, feature2?)
-and then, your movielayout, which could be all the pages from movie details to the ticket. You already have them, but they all could be children to one parent

File: src/layouts/RootLayout.tsx [hunk 5]
```diff
@@ -8,8 +8,8 @@ const routes = [
   },
 
   {
-    name: "Genres",
```
Comment (dnmct) [inline src/layouts/RootLayout.tsx]: pls cleanup
route definitions should not go here

File: src/layouts/MovieDetailLayout.tsx [hunk 6]
```diff
@@ -1,40 +1,37 @@
...
-//     name: "Ticket",
-//     path: "/date&time/selectseats/ticket",
-//   },
-// ]
-
-
-
+const routesMovieDetail = [
+  {
+    name: "Movie Detail",
```
Comment (dnmct) [inline src/layouts/MovieDetailLayout.tsx:5]: pls clean up
route definitions should not go here

File: src/main.tsx [hunk 7]
```diff
@@ -65,8 +65,8 @@ const router = createBrowserRouter([
         path: "home",
       },
       {
-        element: <Genres />,
```
Comment (dnmct) [inline src/main.tsx]: You're not using layouts here

structure should be something like this

```
| /login
| / <NavigationLayout />
  -- index
  -- movies
| /movie <BookingLayout />
  -- :movieId
  -- :movieId/book-time
  -- :movieId/book-seats
  -- :movieId/success 
  ```

**Review by dnmct — CHANGES_REQUESTED** [review]

File: src/layouts/RootLayout.tsx (hunk 5 above)
Comment (peelerzin) [reply src/layouts/RootLayout.tsx]: deleted

**Review by dnmct — APPROVED** [review]

**Pull request merged**